.gitignore

```diff
@@ -20,4 +20,4 @@ yarn-error.log*
 
 
 #TODO add .env to your gitignore
-
+.env
```

Comment:
The file is located in server/.env

Reply:
You can actually ignore this comment - this will gitignore any file named .env (see https://www.atlassian.com/git/tutorials/saving-changes/gitignore)